sdk/trace/src/test/java/io/opentelemetry/sdk/trace/config/SpanLimitsTest.java

```diff
@@ -46,23 +46,23 @@ void updateSpanLimits_All() {
 
   @Test
   void invalidSpanLimits() {
-    assertThatThrownBy(() -> SpanLimits.builder().setMaxNumberOfAttributes(0))
+    assertThatThrownBy(() -> SpanLimits.builder().setMaxNumberOfAttributes(-1))
```

Comment:
Each of these test cases now duplicates a test case directly below it. Same comment as [this](https://github.com/open-telemetry/opentelemetry-java/pull/7030/files#r1931384254).